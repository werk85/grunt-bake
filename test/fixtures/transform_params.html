<span>{{ content.string | repeat:4 }}</span>
<span>{{ content.string | replace:'B':'C' }}</span>
<span>{{ content.array | max:2 | join:' | ' }}</span>