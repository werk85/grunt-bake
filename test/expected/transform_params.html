<span>BakeBakeBakeBake</span>
<span>Cake</span>
<span>Jim | John</span>